packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -186,6 +186,19 @@ function ApolloServerPluginLandingPageDefault(
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <script>
+      var darkMode = window.matchMedia(
+        '(prefers-color-scheme: dark)'
+      ).matches;
+      var link = document.createElement('link');
+      link.rel = 'icon';
+      link.href = \`https://apollo-server-landing-page.cdn.apollographql.com/_latest/assets/favicon$\{darkMode ? '-dark' : ''\}.png\`;
```

Comment:
can we avoid writing the same URL twice?  (could even change the whole thing into a `['icon', 'apple-touch-icon'].forEach`)

also looks like [backticks don't work in IE11](https://caniuse.com/template-literals)... which is not particularly supported (idk if you build the actual bundle in a way that has IE11 support) but seems easy enough to avoid?